js-conditions-loops/conditionsAndloops.js

```diff
@@ -0,0 +1,120 @@
+//First exercise: 
+
+let yearsOfExperience = 5;
+let salary;
+let bonus;
+
+if (yearsOfExperience === 0.5) { 
+    salary = 0;
+    bonus = 0;
+} else if (yearsOfExperience > 0.5 && yearsOfExperience <=1) { 
+    salary = 100;
+    bonus = 0.01 * salary;
+} else if (yearsOfExperience > 1 && yearsOfExperience <=2) { 
+    salary = 333;
+    bonus = 0.03 * salary; 
+}
+else if (yearsOfExperience > 2 && yearsOfExperience <=4) { 
+    salary = 555;
+    bonus = 0.04 * salary; 
+}
+else if (yearsOfExperience >=4) { 
+    salary = 666;
+    bonus = yearsOfExperience * salary; 
+}
+let totalAmount = salary + bonus;
+console.log('Total amount:',totalAmount)
+
+//Second exercise: 
+
+let month = '12';
+
+switch (month) {
+    case '01':
+        message = 'January';
+        break;
+    case '02': 
+        message = 'February';
+        break;
+    case '03': 
+        message = 'March'; 
+        break;
+    case '04': 
+        message = 'April';
+        break;
+    case '05': 
+        message = 'May';
+        break;
+    case '06':
+        message =  'June';
+        break;
+    case '07':
+        message =  'July';
+        break;
+    case '08': 
+        message = 'August';
+        break;
+    case '09':
+        message = 'September';
+        break;
+    case '10': 
+        message ='October';
+        break;
+    case '11': 
+         message = 'November';
+        break;  
+    case '12': 
+         message = 'December';
+        break;     
+    default: 'You are in another galaxy'
+        break;
+}
+
+console.log(message);
+
+//Third exercise
+
+const number = 30;
+
+for (let i = 1; i<=30; i++){
+    if (i % 2!== 0){
+        continue;
+    }
+    if (i === 22) {
+        continue;
+    }
```

Comment:
```suggestion
    if (i % 2!== 0 || i === 22){
        continue;
    }
```
Advice, you can merge your conditions into one line, there will be fewer rows with code